dxDiagram: Update API descriptions (#1384)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/fullScreen.md

Specifies whether or not to display the widget in full-screen mode.

---

> If the **Diagram** widget is contained in a frame, the **fullScreen** option can be set in a user-generated event handler (button click, a key press) because of security reasons.

Reviewer: ```suggestion
If the **Diagram** widget is contained in a frame, the **fullScreen** option can be set in a user-generated event handler (a button click, a key press) for security reasons.
```
